### DISCUSSION TITLE
OOP2_more (27da3f5)

### DISCUSSION THREAD

**Discussion by izywizy** [NightscoutFoundation/xDrip#2950] [Q&A] [answered]
Where to down load this version please ?

**Comment by Navid200** [comment]
Have you checked this?
https://xdrip.readthedocs.io/en/latest/use/OOP/

**Reply by izywizy** [reply]
Yes it's listed there but not downloadable.

**Reply by Navid200** [reply]
@psonnera

**Reply by psonnera** [reply]
It is downloadable. You need to be signed in Google though. Use Chrome.

**Reply by izywizy** [reply]
Thanks for the reply.

So using chrome to go to the page and there is no option to login using google or any other method.

I worked backwards and was eventually offered an option to login to github which was successful using the same login as here but still no way to download.

I am logged into to google

Surely it can't be this difficult!

**Reply by psonnera** [reply, marked as answer]
using the link directly? https://drive.google.com/file/d/1f1VHW2I8w7Xe3kSQqdaY3kihPLs47ILS/view or copy/paste it?

**Reply by izywizy** [reply]
Got it, thanks very much :)

**Reply by PhilTU** [reply]
the "login" refers to your device or chrome browser being logged into your Google account. Check in top right corner of browser.

**Reply by izywizy** [reply]
As I said above I was logged in to Google.
